tasks/6_reverse-string/reverse-string.js

```diff
@@ -0,0 +1,5 @@
+export function reverseString(stringToReverse) {
+  stringToReverse = stringToReverse.split('');
+  stringToReverse.reverse();
+  return stringToReverse.join('');
+}
```

Comment:
it is okay, but you can chain methods `str.split('').reverse().join('');` when you have to long chain you can write every method from new line
```
srt.split('')
  .reverse()
  .join('');
```